lib/CarControl/CarControl.cpp

```diff
@@ -218,7 +218,7 @@ void CarControl::adjust_paddles(int seconds) {
   } else {
     engineerDisplay.print(s.c_str());
   }
-  // SAFTY
+  //#SAFTY#
```

Comment:
Only a minor thing, but since i have seen this in many places: The correct term is: `safety`